### ISSUE TITLE
Line number linking issue with safari browser

### ISSUE THREAD

**Issue by sreehari83** [oracle/opengrok#2296]
once a file is accessed at a given line number, subsequent accesses of that same file incorrectly provide a hyperlink to that same old line number and not to the newly requested line number.

ex:
http://demo.opengrok.org/search?project=elasticsearch&q=&defs=&refs=%24group_issues&path=&hist=&type=

It gives the g
 <b > 99 </b> my $group_issues = $issues->{$group} or next;
<b> 106 </b> for my $header ( sort keys %$group_issues ) {
<b> 107 </b> my $header_issues = $group_issues->{$header}; 

First click on 106 line number and it is showing correct line number in file.
and then click on any line number 99/107 , It is showing always 106 line number in file.

This problem with safari browser

**Comment by vladak** [comment]
To be precise, this happens after you go back to the search result page and click the different lines ?

What Safari version are you running ?

This does not happen for me on FF 61.0.1.

**Comment by sreehari83** [comment]
yes, this is happening after go back to the search result page and click on the different lines .

this problem only with safari only.

My safari Version is 11.1.2 (12605.3.8.1)

**Comment by sreehari83** [comment]
its working fine on opengrok-0.13-rc1 with safari

**Comment by sreehari83** [comment]
any update on this issue

**Comment by vladak** [comment]
If you could isolate this to particular release candidate that would be nice start to get this fixed (assuming this is problem in OpenGrok).
